cargo-pgx/src/command/package.rs

```diff
@@ -89,15 +92,16 @@ impl CommandExecute for Package {
             &pg_config,
             out_dir,
             &profile,
+            &self.cross_args,
             self.test,
             &self.features,
         )
     }
 }
 
 #[tracing::instrument(level = "error", skip_all, fields(
-    pg_version = %pg_config.version()?,
-    profile = ?profile,
+    pg_version = % pg_config.version() ?,
```

Comment:
Nit: these whitespace changes aren't necessary.

Reply:
I'll need to double check, but this probably got changed by `rustfmt`. Or maybe by that idiotic CLion formatter (why it doesn't default to `rustfmt` is beyond me).